Intro background + ssle (#51)
AAU-Dat/P9-ZKP: report/src/sections/07-future-works.tex

> \section{Future works}\label{sec:future-works}

Reviewer: ```suggestion
\section{Future work}\label{sec:future-works}
```

> This is a potential discussion section.

Reviewer: ```suggestion

```

\subsection{potential mitigations}\label{subsec:potential-mitigations}

> At the moment there exists an improvement proposal to include a Secret single leader election (SSLE) mechanism called Whisk in Ethereum~\cite{EthereumResearchSSLE2024}.

Reviewer: ```suggestion
At the moment there exists an improvement proposal to include a Secret Single Leader Election (SSLE) mechanism called Whisk in Ethereum~\cite{EthereumResearchSSLE2024}.
```

This method aims to improve the security of the network by obfuscating the identity of the proposer.
This would make it harder for an adversary to perform the Proposer DoS attack since the adversary would not know which validator to target.
But even if this stops the DoS attack, it does not prevent the de-anonymization of the validators.

> But that in itself is a good reason to look implement SSlE in Ethereum as a further step to improve the security of the network.

Reviewer: ```suggestion
But hindering a DoS attack in itself is a good reason to look at implementing SSLE in Ethereum as a further step to improve the security of the network.
```
